lib/node_modules/@stdlib/math/base/special/xlog1py/README.md

```diff
@@ -101,7 +101,7 @@ for ( i = 0; i < 100; i++ ) {
 ## See Also
 
 -   <span class="package-name">[`@stdlib/math/base/special/log1p`][@stdlib/math/base/special/log1p]</span><span class="delimiter">: </span><span class="description">evaluate the natural logarithm of 1+x.</span>
--   <span class="package-name">[`@stdlib/math/base/special/xlogy`][@stdlib/math/base/special/xlogy]</span><span class="delimiter">: </span><span class="description">compute `x * ln(y)` so that the result is `0` if `x = 0`.</span>
+-   <span class="package-name">[`@stdlib/math/base/special/xlogy`][@stdlib/math/base/special/xlogy]</span><span class="delimiter">: </span><span class="description">compute `x \* ln(y)` so that the result is `0` if `x = 0`.</span>
```

Comment:
@Planeshifter This was exactly what I was alluding to in our prior discussion. The asterisk should not be escaped. See https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/xlog1py/README.md#see-also.

Reply:
@kgryte Addressed.